helios-testing/src/main/java/com/spotify/helios/testing/HeliosSoloDeployment.java

```diff
@@ -74,18 +85,19 @@
  * master and one Helios agent deployed in Docker. Helios Solo uses the Docker instance it is
  * deployed on to run its jobs.
  */
-public class HeliosSoloDeployment implements HeliosDeployment {
+class HeliosSoloDeployment implements HeliosDeployment {
```

Comment:
Yes, the idea is for users to not to use `HeliosSoloDeployment` at all. I'm also planning on making breaking changes to remove the innumerable configuration options.